### ISSUE TITLE
Some channels cannot be added to a category

### ISSUE THREAD

**Issue by ilanshamir** [mattermost/mattermost#15545]
#### Summary
Some channels cannot be added to a category

#### Steps to reproduce
Using the latest server build from Sep-19-2021 and the Windows client 4.5.4

#### Expected behavior
I create a category and add channels to it
Some channels are added, but others will consistently fail to be added
They will move into the category for a brief second and then go back to their original locaiton

#### Observed behavior (that appears unintentional) 
The channel disappers for a second and then shows up again.
It doesn't move into the category.
See attached gif

![builds-Resilio-Inc-Chat-2020-09](https://user-images.githubusercontent.com/19560106/93836829-d202fa00-fc38-11ea-9ecc-cc64f72e4d43.gif)

#### Possible fixes
Couldn't find a workaround

**Comment by ahmaddanialmohd** [comment]
Hello, @ilanshamir 

I attempted to reproduce this in the following environment to no avail:

* Mattermost Server 5.27.0
* Mattermost Mac Desktop App 4.5.0

I also tried using the same channel name format as yours `- [O] Something` but I was still able to move the channels across  different categories using *Move to > Category* as well as drag and drop.

A couple of questions that I would like to clarify from your end:

* Since you mentioned that the behavior is inconsistent based on the statement below, under what circumstances do the category move fails?
> Some channels are added, but others will consistently fail to be added
* Does the problem occur when you move to other categories besides *My POCs*?
* Is this also happening on the web browser on Incognito / private mode?
* Have you tried to clear the cache of the desktop app and then try to move the channel again?
* Does the process fail even when you attempt to drag and drop the channel to the category of your choice?
